### PR TITLE
implemented RSSI code

### PR DESCRIPTION
**Pull request by SnackkOverflowError**

### CODE REVIEW

File: Safety/Src/RSSI.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include "RSSI.hpp"
+#include "gpio.h"
+
+static volatile bool failed = false;
+static volatile int16_t counter = 0;
+
+bool CommsFailed() {
```
Comment (Pop0097) [inline Safety/Src/RSSI.cpp:7]: In the future methods like this should start with "get"

File: Safety/Src/RSSI.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#include "RSSI.hpp"
+#include "gpio.h"
+
+static volatile bool failed = false;
+static volatile int16_t counter = 0;
```
Comment (Pop0097) [inline Safety/Src/RSSI.cpp:5]: uint16

**Review by Fberbari — APPROVED** [review]
I'm fine with this

File: Safety/Inc/RSSI.hpp [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#ifndef RSSI_HPP
+#define RSSI_HPP
+
+bool CommsFailed();
```
Comment (Pop0097) [inline Safety/Inc/RSSI.hpp:4]: Comments over functions to explain purpose

**Review by Pop0097 — APPROVED** [review]
Looks good. Comments I put should be implemented after competition.

**Pull request merged**